config_gen.py

```diff
@@ -303,6 +289,42 @@ def run(cmd, *args, **kwargs):
         print("\nRunning make...")
         run(make_args, env=env, **proc_opts)
 
+    elif(os.path.exists(os.path.join(project_dir, "Make/options"))):
+        # Added for supporting wmake (OpenFOAM build system) 2016/03/18
+        print("Found OpenFOAM Make/options")
+        print("Preparing build directory...")
+        build_dir = tempfile.mkdtemp()
+
+        # wmake uses environment variables starting with WM_
+        # since OpenFOAM 3.x.x, there were some variables added.
+        keys = os.environ.keys()
+        WM = re.compile('^WM_')
+        for key in keys:
+            if(WM.match(key)):
+                env[key] = os.environ[key]
+
+        make_args = ['wmake']
+       
+        # Since clang compilation has been failed with intel compiler,
+        # set environmental variables to gcc to make fake_build operate normally.
+        
+        env['WM_COMPILER']='Gcc'
+        env['WM_CC']='gcc'
+        env['WM_CXX']='g++'
+
+        # wmake won't work well with FNULL
```

Comment:
At the time I tested my code, the code failed to work correctly without None specified. But, now the code is working. So those lines are removed.